src/rovr_control/package.xml

```diff
@@ -32,6 +32,7 @@
   <exec_depend>rovr_interfaces</exec_depend>
   <exec_depend>ros2socketcan_bridge</exec_depend>
   <exec_depend>python3-serial</exec_depend>
+  <exec_depend>nav2_simple_commander</exec_depend>
```

Comment:
What did we edit to warrant this?

Reply:
Nothing in this PR. I realized that this line should have been added in a previous PR and wasn't so I fixed it now.